keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/ImageHandler.kt

```diff
@@ -169,6 +177,16 @@ class ImageHandler(
         parameters = mapOf("delta" to diff.toDebug())
       )
       publisher.publishEvent(BakeLaunched(desiredVersion))
```

Comment:
This event, kicked off in the image handler, is our cue to start monitoring the bake. It is the first event in the series of events for bakes. We assume that a `NOT_STARTED` event is always the first in the chain.